packages/git-diff/lib/main.js

```diff
@@ -1,32 +1,52 @@
-const GitDiffView = require('./git-diff-view');
-const DiffListView = require('./diff-list-view');
+'use babel';
+
+import { CompositeDisposable } from 'atom';
+import GitDiffView from './git-diff-view';
+import DiffListView from './diff-list-view';
 
 let diffListView = null;
+let diffViews = null;
+let subscriptions = null;
+
+export default {
+  activate(state) {
+    subscriptions = new CompositeDisposable();
+    diffViews = new Set();
+
+    subscriptions.add(
+      atom.workspace.observeTextEditors(editor => {
+        const editorElement = atom.views.getView(editor);
+        const diffView = new GitDiffView(editor);
 
-module.exports = {
-  activate() {
-    const watchedEditors = new WeakSet();
-
-    atom.workspace.observeTextEditors(editor => {
-      if (watchedEditors.has(editor)) return;
-
-      new GitDiffView(editor).start();
-      atom.commands.add(
-        atom.views.getView(editor),
-        'git-diff:toggle-diff-list',
-        () => {
-          if (diffListView == null) diffListView = new DiffListView();
-          diffListView.toggle();
-        }
-      );
-
-      watchedEditors.add(editor);
-      editor.onDidDestroy(() => watchedEditors.delete(editor));
-    });
+        diffViews.add(diffView);
+
+        let editorSubs;
```

Comment:
Just assign it directly if you need it somewhere else. See https://github.com/atom/atom/pull/21968/files#r578696295